commit cec8eeb69dc65eb91e0cb23ebff23530d9397425
Author: Jaipaul Cheernam <jaipaul.cheernam@est.tech>
Date:   Sat Aug 15 14:00:36 2026 -0400

    getport: fix missing stddef.h inclusion
    
    getport.c uses offsetof() but does not include <stddef.h>. On glibc
    this works because offsetof is pulled in transitively, but on musl it
    is not available without the explicit include, causing:
    
      getport.c:459:41: error: implicit declaration of function 'offsetof'
    
    Signed-off-by: Jaipaul Cheernam <jaipaul.cheernam@est.tech>
    Signed-off-by: Steve Dickson <steved@redhat.com>

diff --git a/support/nfs/getport.c b/support/nfs/getport.c
index 608e185b..6bd2f713 100644
--- a/support/nfs/getport.c
+++ b/support/nfs/getport.c
@@ -27,6 +27,7 @@
 #endif
 
 #include <sys/types.h>
+#include <stddef.h>
 #include <sys/time.h>
 #include <unistd.h>
 #include <fcntl.h>
